utils/zerovec/Cargo.toml

```diff
@@ -28,6 +28,7 @@ all-features = true
 [dependencies]
 serde = { version = "1.0", optional = true, default-features = false, features = ["alloc"] }
 yoke = { version = "0.4.0", path = "../yoke", optional = true }
+zerofrom = { version = "0.1.0", path = "../zerofrom" }
```

Comment:
Thought: Should this be an optional dependency (but enabled by default)?

Reply:
The VZV custom derive will need it mandatorily I think.

We could design it in an optional way but I think it's small enough to prefer the mandatory dep.